WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -451,6 +451,31 @@ private void newPostSetup() {
         mShortcutUtils.reportShortcutUsed(Shortcut.CREATE_NEW_POST);
     }
 
+    private void newPageFromLayoutPickerSetup(String title, String content) {
```

Comment:
We can probably reuse the implementation of `newPostSetup` since the only difference is the `instantiatePostModel` call.
This is just a suggestion and I understand that the way that it is implemented now might be more clear and easier to extend in the future.